bundles/org.openhab.binding.digiplex/src/main/resources/OH-INF/thing/area.xml

```diff
@@ -109,6 +109,10 @@
 		<item-type>Contact</item-type>
 		<label>Area in Alarm</label>
 		<description>Indicates if area is in alarm</description>
+		<tags>
+			<tag>Alarm</tag>
+			<tag>Presence</tag>
```

Comment:
Little doubt here as the area maybe in alarm due to presence, but also vibration, motion, flooding, fire, etc. When looking at the SemanticTags.csv i can;t find a clear suggestion.

Reply:
Then let's just use bare Alarm.